var/spack/repos/builtin/packages/migraphx/package.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Migraphx(CMakePackage):
+    """ AMD's graph optimization engine."""
+
+    homepage = "https://github.com/ROCmSoftwarePlatform/AMDMIGraphX"
+
+    maintainers = ['srekolam', 'arjun-raj-kuppala']
+
+    def url_for_version(self, version):
+        # Fix up a typo in the 3.5.0 release.
+        if version == Version('3.5.0'):
+            return "https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/archive/rocm-3.5.0.tar.gz"
+        if version == Version('3.7.0'):
+            return "https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/archive/0.7.tar.gz"
+        if version == Version('3.8.0'):
+            return "https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/archive/0.8.tar.gz"
```

Comment:
```suggestion
        url = "https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/archive/{0}.tar.gz"
        if version > Version('3.5'):
            return url.format(version)
        else:
            return url.format('rocm-' + version)
```

Reply:
done. i have updated the file with the requested changes